python/cdp/evm_smart_account.py

```diff
@@ -693,6 +693,30 @@ async def sign_typed_data(
             ),
         )
 
+    async def __experimental_use_network(self, network: str):
```

Comment:
Looks like double_underscore has a special meaning in Python – this needs to be double_underscored on both sides, so:
```
async def __experimental_use_network__(
```